settings.xml

```diff
@@ -0,0 +1,26 @@
+<settings>
+<servers>
+  <server>
+    <id>maven.oracle.com</id>
+    <username></username>
+    <password></password>
+    <configuration>
+      <basicAuthScope>
+        <host>ANY </host>
+        <port>ANY </port>
```

Comment:
I expect this should make a difference, but I'm not sure, as it looks like it worked for you as it is now.
```suggestion
        <host>ANY</host>
        <port>ANY</port>
```

Reply:
Wonder where those stray space symbols came from... Thanks, will fix